docs/faq/terminology.rst

```diff
@@ -0,0 +1,46 @@
+.. _Observation_Terminology:
+
+***********
+Terminology
+***********
+
+Scheduling
+==========
+
+Scheduling observations is a common task in astronomy, but there is a lack of
+formal terminology. Here we define the terminology as it will be used in astroplan
+for scheduling tasks. This is for consistency within the code and the documentation.
+
+The terms used are as follows:
+
+* **visit**: a period of time spent observing a single target
+* **observing block** (OB): the minimum schedulable block (tentative: a group of visits)
```

Comment:
Just a comment from the sidelines: I'm surprised that there isn't a separate MSB in addition to an OB. I think in Erik's example above would work as two separate OBs (due to the filter difference), or one big one. I think it's clearer to have the first and join them into an MSB, but one can argue the other way around.
However consider the case when you do a photometric survey when one data point is no data point, so you want to have the exact same OB repeated over a certain number of times to form an MSB.